README.md

```diff
@@ -13,6 +13,9 @@ This command merges all transactions previously saved into a single file to be p
 ### Generating reports
 Reports are generated for a given time period selected using two parameters `--start-time <ISO8061 DATETIME>` and `--end-time <ISO8061 DATETIME>`.
 
+> It's a must for generating report that both datetime entries should include time and timezone, following the ISO8061 format, eg: 
```

Comment:
nitpick: this sounds redundant, ISO8061 is already mentioned and you added an example below.

Reply:
I agree